src/lib/commonPropTypes.ts

```diff
@@ -17,7 +17,11 @@ export const commonUIComponentPropTypes = {
   className: PropTypes.string,
 }
 
-export const contentComponentPropsTypes = {
+export const contentShorthandComponentPropsTypes = {
+  content: customPropTypes.itemShorthand,
+}
+
+export const contentNodeComponentPropsTypes = {
   content: customPropTypes.contentShorthand,
```

Comment:
would suggest to rename this custom prop type - while underlying type is correct, its name is misleading. Something like `nodeContent` would be a better choice here: in either case, I would rather not expect `shorthand` word being used here